src/pages/Integrations/SplunkSetup/SplunkSetupPage.tsx

```diff
@@ -0,0 +1,366 @@
+import './SplunkSetup.scss';
+
+import {
+    ActionGroup,
+    Alert,
+    Button,
+    CodeBlock,
+    CodeBlockCode,
+    Flex,
+    FlexItem,
+    Form,
+    FormGroup,
+    Popover,
+    TextInput
+} from '@patternfly/react-core';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import { Main, PageHeader, PageHeaderTitle, Section } from '@redhat-cloud-services/frontend-components';
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
```

Comment:
What do we need Redux for? Can React states be used instead?

Reply:
FWIW we don't use redux here except for connecting with the toasts-api from chrome.

Reply:
Removing...